docs/src/pages/customization/css-in-js.md

```diff
@@ -104,14 +141,43 @@ import JssProvider from 'react-jss/lib/JssProvider';
 import { create } from 'jss';
 import { createGenerateClassName, jssPreset } from 'material-ui/styles';
 
+const generateClassName = createGenerateClassName();
+const jss = create(jssPreset());
+// We define a custom insertion point that JSS will look for injecting the styles in the DOM.
+jss.options.insertionPoint = 'jss-insertion-point';
```

Comment:
If you provide a string jss will only match with a Comment Node.

https://github.com/cssinjs/jss/blob/8252b25dfa7ebdb9ab9112c6760279cd79e742d2/src/renderers/DomRenderer.js#L214-L215

You need to provide the element itself: `document.getElementById('jss-insertion-point')`